hw/top_earlgrey/data/top_earlgrey.hjson

```diff
@@ -204,7 +204,17 @@
         },
       ],
     }
-
+    { name: "rv_dm",
```

Comment:
ah! glad someone one is using the "host" structure!

Reply:
we are only....a few PRs away from fully automating the host instantiation! probably another day...

Reply:
jep ;)